qr-code-auth/README.md

```diff
@@ -0,0 +1,277 @@
+# Collection - QrCodeAuth
+
+## API Endpoints
+
+|                                       Route                                        |                                         Description                                          |
+| :--------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------: |
+|                      [POST /qr-code-auth](#post-qr-code-auth)                      |                            Creates a new authentication document.                            |
+|                       [GET /qr-code-auth](#get-qr-code-auth)                       |           Retrieves all the authentication document belonging to a specified user            |
+| [PATCH /qr-code-auth](#patch-qr-code-authauthorization_statusauthorization_status) | Updates the authorization_status field of an existing qr-code-auth document for the specified user. |
+
+## POST /qr-code-auth
+
+    Creates a new qr-code-auth document.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **user_id** (required, string): Specifies the user id for the document.
+    - **device_info** (required, string): Specifies the device information associated with the authentication.
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  None
+- **Success Response:**
+  - **Code:** 201
+    - **Content:**
+      ```json
+      {
+        "message": "String",
+        "data": {
+          "user_id": "String",
+          "device_info": "String",
+          "authorization_status": "String",
+          "access_token": "String"
+        }
+      }
+      ```
+- **Error Response:**
+
+  - **Code:** 400
+
+    - **Content:**
+      ```json
+      {
+        "message": "Bad Request"
+      }
+      ```
+
+  - **Code:** 409
+
+    - **Content:**
+      ```json
+      {
+        "message": "The authentication document has already been created"
+      }
+      ```
+
+  - **Code:** 500
+    - **Content:**
+      ```json
+      {
+        "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+      }
+      ```
+
+- **Example for user authentication document creation request:**
+  POST /qr-code-auth<br/>
+  Content-Type: application/json<br/>
+  Request-Body:<br/>
+
+  ```json
+  {
+    "device_info": "t5k77PHnuDSrgEzvMJAj",
+    "user_id": "NLFSj7Kz30oHgolfIZtJ"
+  }
+  ```
+
+  Response :
+  Status 201<br/>
+  Content-Type: application/json<br/>
+
+  ```json
+  {
+    "data": {
+      "user_id": "SooJK37gzjIZfFNH0tlL",
+      "device_info": "t5k77PHnuDSrgEzvMJAj",
+      "authorization_status": "NOT_INIT",
+      "access_token": ""
```

Comment:
from where do we get the `accesstoken`, according to the code we are returning `githubaccesstoken`, is this the same?

Reply:
no this is not the same as `githubaccesstoken`. I think this is still not decided how will this be modified or saved. @shreya-mishra correct me if I am wrong.

Reply:
we are generating from the (JWT) backend

Reply:
please check the flow once in the issue ticket

Reply:
can u share the issue ticket?

Reply:
linking it here for future ref:
https://github.com/Real-Dev-Squad/mobile-app/issues/146